gsoc/background.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title:  "Background on GSoC: start here!"
+show_main: false 
+---
+
+# Background: Start Here!
+
+### What is GSoC? 
+
+Google Summer of Code is a remote summer coding internship program funded by 
+Google. Google distributes funds to open software development organizations to 
+recruit new developers and mentors for summer internships. You get paid to write 
+software at home (or wherever you like), with guidance from expert mentors 
+distributed around the globe.
+
+To apply to be a GSoC intern, you must write an application to GSoC and an 
```

Comment:
while it's really an internship like thing, it's officially not an intership so we should try to avoid calling it such. What about changing intern to student here?

Reply:
Also, there is only one application, so what about this:

To apply to be a GSoC student, you must write an application to GSoC ideally after you have discussed your project plan with your potential mentors.

Reply:
@bsipocz - what do you mean with *there is only one application*? The student can [submit up to three applications](https://google.github.io/gsocguides/student/writing-a-proposal.html)

Reply:
oh, what I meant is that there is only one application per project, no need to apply to google *and* to the org. Whether the students can have multiple application with different projects that's something different.